2016/proposals/patrick-kofod-mogensen.md

```diff
@@ -0,0 +1,56 @@
+# Making Optim.jl great again
```

Comment:
Couldn't think of a better title, and thought I would cater to the American reviewers. Good thing I have more time to brain storm this particular part. Unless I misread your air horn, and it is one of the rare "I applaud this"-air horns.